apps/api/v1/pages/api/availabilities/mentors/_get.ts

```diff
@@ -0,0 +1,59 @@
+import type { NextApiRequest } from "next";
+
+import { HttpError } from "@calcom/lib/http-error";
+import { defaultResponder } from "@calcom/lib/server";
+import prisma from "@calcom/prisma";
+
+import {
+  schemaAvailabilityCreateBodyParams,
+  schemaAvailabilityEditBodyParams,
+} from "~/lib/validations/availability";
+import { schemaUserReadPublic } from "~/lib/validations/user";
+
+async function getHandler(req: NextApiRequest) {
+  const {
+    body,
```

Comment:
Why not use query parameters? Using body in a get request is unorthodox to say the least. 

Other API examples from the same subfolder to use query parameters are as follows

apps/api/v1/pages/api/availability/_get.ts
apps/api/v1/pages/api/bookings/_get.ts